src/roles/vsd-deploy/tasks/renew_ejabberd_license.yml

```diff
@@ -0,0 +1,55 @@
+- block:
```

Comment:
Blocks should be named - according to Ansible-lint.

- name: Run when license file is defined
  block: